README.md

```diff
@@ -1,60 +1,48 @@
-from (pry):3:in `__pry__`
-[4] pry(main)> require 'better_html/parser'
-=> true
-[5] pry(main)> BetterHtml::Parser
-=> BetterHtml::Parser
-```
+# Tapioca - The swiss army knife of RBI generation
 
-In order to make sure that `tapioca` can reflect on that type, we need to add the line `require "better_html/parser"` to the `sorbet/tapioca/require.rb` file. This will make sure `BetterHtml::Parser` is loaded into memory and a type annotation is generated for it in the `better_html.rbi` file. If this extra `require` line is not added to `sorbet/tapioca/require.rb` file, then the definition for that type will be missing from the RBI file.
-
-If you ever run into a case, where you add a gem or update the version of a gem and run `tapioca gem` but don't have some types you expect in the generated gem RBI files, you will need to make sure you have added the necessary requires to the `sorbet/tapioca/require.rb` file.
-
-You can use the command `tapioca require` to auto-populate the `sorbet/tapioca/require.rb` file with all the requires found
-in your application. Once the file generated, you should review it, remove all unnecessary requires and commit it.
-
-## How does tapioca compare to "srb rbi gems" ?
+![Build Status](https://github.com/Shopify/tapioca/workflows/CI/badge.svg)
 
-[Please see the detailed answer on our wiki](https://github.com/Shopify/tapioca/wiki/How-does-tapioca-compare-to-%22srb-rbi-gems%22-%3F)
+Tapioca makes it easy to work with [Sorbet](https://sorbet.org) in your codebase. It surfaces types and methods from many sources that Sorbet cannot otherwise see – such as gems, Rails and other DSLs – compiles them into [RBI files](https://sorbet.org/docs/rbi) and makes it easy for you to add gradual typing to your application.
+
+**Features**:
+
+* Easy installation and configuration
+* Generation of RBI files for the gems used in your application
+  * Automatic generation from your application's Gemfile
+  * Signatures importation from the gems source code
+  * Documentation importation from the gems source code
+  * Synchronization validation for your CI
+* Generation of RBI files for various DSL patterns that relies on meta-programming
+  * Automatic generation from your application's content
+  * Support many DSL patterns such as Rails, Google Protobuf, SmartProperties and more out of the box
+  * Extensible interface that allows you to write your own DSL compilers for other DSL patterns
+  * Automatic generation of signatures for methods from known DSLs
+  * Synchronization validation for your CI
+* Management of shim RBI files
+  * Find useless definitions in shim RBI files from gems generated RBI files
+  * Find useless definitions in shim RBI files from DSL generated RBI files
+  * Find useless definitions in shim RBI files from Sorbet's embedded RBI for core and stdlib
+  * Synchronization validation for your CI
 
 ## Installation
 
 Add this line to your application's `Gemfile`:
 
-```ruby
+~~~rb
```

Comment:
Just curious: any difference between using a "`" or a "~"?

Reply:
None as long as the style is consistent, I put back ``` everywhere 👍